klio get fails when klio.yaml is empty

When klio.yaml is empty, `klio get` fails with following error:

```
[INFO] Installed deploy@0.8.1 from file:///Users/qzb/Workspace/klio/registry.yaml as deploy
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x1389186]

goroutine 1 [running]:
gopkg.in/yaml%2ev3.handleErr(0xc0002370e0)
        /Users/qzb/.go/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200615113413-eeeca48fe776/yaml.go:294 +0x85
panic(0x1462f60, 0x17ac0c0)
        /usr/local/opt/go/libexec/src/runtime/panic.go:969 +0x175
github.com/g2a-com/klio/internal/schema.ProjectConfig.MarshalYAML(0xc000238750, 0x23, 0x1, 0x0, 0x0, 0xc000020230, 0x1, 0x1, 0x0, 0x0, ...)
        /Users/qzb/Workspace/klio/internal/schema/schema.go:62 +0x46
gopkg.in/yaml%2ev3.(*encoder).marshal(0xc00035a000, 0x0, 0x0, 0x1465200, 0xc0000c3310, 0x16)
        /Users/qzb/.go/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200615113413-eeeca48fe776/encode.go:135 +0x196
gopkg.in/yaml%2ev3.(*encoder).marshalDoc(0xc00035a000, 0x0, 0x0, 0x1465200, 0xc0000c3310, 0x16)
        /Users/qzb/.go/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200615113413-eeeca48fe776/encode.go:105 +0x137
gopkg.in/yaml%2ev3.(*Encoder).Encode(0xc000237138, 0x1465200, 0xc0000c3310, 0x0, 0x0)
        /Users/qzb/.go/pkg/mod/gopkg.in/yaml.v3@v3.0.0-20200615113413-eeeca48fe776/yaml.go:251 +0xea
github.com/g2a-com/klio/internal/schema.SaveConfigFile(0x1465200, 0xc0000c3310, 0xc000238750, 0x23, 0x1, 0xc000020230)
        /Users/qzb/Workspace/klio/internal/schema/util.go:72 +0x4cd
github.com/g2a-com/klio/internal/schema.SaveProjectConfig(...)
        /Users/qzb/Workspace/klio/internal/schema/util.go:99
github.com/g2a-com/klio/internal/cmd/get.run(0x14cc754, 0x4, 0x0, 0x0, 0x0, 0x0, 0xc000241190, 0x9, 0xc000241188, 0x5, ...)
        /Users/qzb/Workspace/klio/internal/cmd/get/get.go:126 +0x3d0
github.com/g2a-com/klio/internal/cmd/get.NewCommand.func1(0xc000243680, 0xc0001e7830, 0x1, 0x3)
        /Users/qzb/Workspace/klio/internal/cmd/get/get.go:31 +0xc5
github.com/spf13/cobra.(*Command).execute(0xc000243680, 0xc0001e77a0, 0x3, 0x3, 0xc000243680, 0xc0001e77a0)
        /Users/qzb/.go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xc000243400, 0x4, 0x0, 0x0)
        /Users/qzb/.go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2fe
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/qzb/.go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/g2a-com/klio/pkg/cli.(*CLI).Execute(0xc000237f38, 0x1775520, 0xc000000180)
        /Users/qzb/Workspace/klio/pkg/cli/cli.go:38 +0x389
main.main()
        /Users/qzb/Workspace/klio/cmd/klio/klio.go:17 +0x73
```

This error doesn't occur when klio.yaml contains dependencies property:

```yaml
dependencies: {}
```

--- a/internal/schema/schema.go
+++ b/internal/schema/schema.go
@@ -58,6 +58,10 @@ func (p ProjectConfig) MarshalYAML() (interface{}, error) {
 	var defaultRegistryValueNode *yaml.Node
 	var dependenciesValueNode *yaml.Node
 
+	if p.yaml == nil {
+		return nil, errors.New("missing dependencies or defaultRegistry key")
+	}
+
 	// Find nodes to encode
 	for i := 0; i < len(p.yaml.Content)/2; i++ {
 		k := p.yaml.Content[i*2]

--- a/internal/schema/schema.go
+++ b/internal/schema/schema.go
@@ -59,7 +59,7 @@ func (p ProjectConfig) MarshalYAML() (interface{}, error) {
 	var dependenciesValueNode *yaml.Node
 
 	if p.yaml == nil {
-		return nil, errors.New("missing dependencies or defaultRegistry key")
+		return minimalKlioFile(), nil
 	}
 
 	// Find nodes to encode
@@ -111,6 +111,24 @@ func (p ProjectConfig) MarshalYAML() (interface{}, error) {
 
 	// Return result
 	return p.yaml, nil
+}
+
+func minimalKlioFile() *yaml.Node {
+	return &yaml.Node{
+		Kind: yaml.MappingNode,
+		Tag:  "!!map",
+		Content: []*yaml.Node{
+			{
+				Kind:  yaml.ScalarNode,
+				Tag:   "!!str",
+				Value: "dependencies",
+			},
+			{
+				Kind: yaml.MappingNode,
+				Tag:  "!!map",
+			},
+		},
+	}
 }
 
 func (p *ProjectConfig) UnmarshalYAML(node *yaml.Node) error {

